### PR TITLE
Update release notes to include qualification and profiling tool

### PR DESCRIPTION
**Pull request by sameerz**
Updating release notes in branch-21.08 as a follow on to PR #2848

### CODE REVIEW

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

**Review by viadea — APPROVED** [review]

File: docs/download.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ New functionality for this release includes:
   * Range windows supporting non-timestamp order by expressions
 * Enabling large joins that can spill out of memory
 * Support for the `concat_ws` operator
+* Spark Qualification and Profiling Tools
+  * A qualification tool to look at a set of applications to determine if the RAPIDS Accelerator for
```
Comment (gerashegalov) [inline docs/download.md:77]: I think there should be some language that it's based on Spark event logs to set the right expectations

**Comment by sameerz** [comment]
> I think there should be some language that it's based on Spark event logs to set the right expectations

This is a brief mention in the release notes, there is more detail in the [tools/README.md](https://github.com/NVIDIA/spark-rapids/blob/branch-21.08/tools/README.md).  In 21.08, @viadea will also add a doc that appears in https://nvidia.github.io/spark-rapids/ to explain these tools in more detail.

**Comment by sameerz** [comment]
build

**Pull request merged**